lib/Parser/rterrors.h

```diff
@@ -20,6 +20,7 @@
 #define RT_ERROR_MSG_UNUSED_ENTRY
 #endif
 
+// NOTE: The VBSERR macros should not be changed due to legacy dependencies outside of Chakra.
```

Comment:
Spoke with @ianwjhalliday offline and workshopped a previous version of the message to communicate the state of things more clearly:

```
// These VBS macros were taken from VBScript and the names and definitions are written in stone.
// Other projects depend on these definitions as well.
// Keep the commented lines as documentation of values that should not be reused.
```